### DISCUSSION TITLE
Empty channel selector and unable to save settings

### DISCUSSION THREAD

**Discussion by Spiraculous** [RaspAP/raspap-webgui#1073] [Q&A] [answered]
Way back at the start of WFH I quickly put together a 'temporary' Pi AP which, of course, is still in use.  It's been so reliable that I pretty much forgot it was there.  The AP is bridge-mode via Ethernet to the main router.

I've just upgraded from (I think) 2.5 to 2.7.1 using the FAQ instructions.  The AP is working fine, is still accepting connections using the old credentials and bridging to the main network.  However the dashboard is displaying no traffic or connection information and the  Hotspot form channel selector is blank - there are no rows in the drop down.  Also, whenever I click Hotspot 'Save Settings' I get

Attempting to set channel to invalid number.
Attempting to set channel outside of permitted range
Unable to save wifi hotspot settings

I see that the AP is actually broadcasting on channel 6.  Country Code is set to United Kingdom.

Any suggestions?

**Comment by billz** [comment, marked as answer]
Re: stats, here's your answer https://github.com/RaspAP/raspap-webgui/releases/tag/2.7.0
We don't get to choose when/how dependencies are updated in RPi OS.

Clearing your browser cache should load the updated channel selector function.

**Reply by Spiraculous** [reply]
Thanks for the very prompt reply, that's fixed the channel selection and save problems.  Still no traffic graph, though as this AP is now a permanent fixture I'm going to re-install starting from Bullseye Lite rather than the Buster Desktop that it went onto a year ago.
